src/types/subscriptions.d.ts

```diff
@@ -139,4 +143,6 @@ export interface SubscriptionsEndpoint
   GetAttachedProducts(id: string) : Promise<Resource<SubscriptionOfferingProduct[]>>
 
   GetAttachedPlans(id: string) : Promise<Resource<SubscriptionOfferingPlan[]>>
+
+  CreateState(id: string, action: SubscriptionsStateAction) : Promise<never>
```

Comment:
What does the new endpoint return?

Reply:
Nothing, actually. I used 'never,' but should I use something else?

Reply:
@rostyk-kanafotskyy  Not 100% sure, but Promise<void> would have a similar effect?

Reply:
I did a little thinking about this and changed `Promise<never>` to `Promise<void>`. This is the right option. 
Thanks for pointing this out!